datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -109,6 +120,9 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
   /** Tracks modifications in order to update the UI. */
   private val modificationCount = MutableStateFlow(0)
 
+  /** Toggles review mode. */
+  private val reviewMode = MutableStateFlow(initialReviewMode)
```

Comment:
end with flow to contrast with `pageFlow`

```suggestion
  private val reviewFlow = MutableStateFlow(initialReviewMode)
```